firebase-perf/src/main/java/com/google/firebase/perf/config/ConfigResolver.java

```diff
@@ -767,6 +768,32 @@ public float getFragmentSamplingRate() {
     return config.getDefault();
   }
 
+  /** Returns if _experiment_app_start_ttid should be captured. */
+  public boolean getIsExperimentTTIDEnabled() {
+    // Order of precedence is:
+    // 1. If the value exists in Android Manifest, return this value.
+    // 2. Cannot read value from RC because it's not initalized yet
+    // 3. If the value exists in device cache, return this value.
+    // 4. Otherwise, return default value.
+    ExperimentTTID config = ExperimentTTID.getInstance();
+
+    // 1. Reads value in Android Manifest (it is set by developers during build time).
+    Optional<Boolean> metadataValue = getMetadataBoolean(config);
+    if (metadataValue.isAvailable()) {
+      return metadataValue.get();
+    }
+
+    // 2. Cannot read value from RC because it's not initialized yet.
```

Comment:
Just to make sure my understanding is correct: for the very first app start of an app, we will read the default value.

Reply:
Yes, for the very very first without any prior RC fetch, it will use default value.